files/default/neutron-ha-tool.py

```diff
@@ -576,6 +600,195 @@ def l3_agent_evacuate(qclient, agent_host, agent_picker, router_filter,
     return errors
 
 
+def dhcp_agent_evacuate(qclient, agent_host, agent_picker,
+                      noop=False, wait_for_network=True, fail_fast=False):
+    """
+    Retreive a list of networks scheduled on the listed DHCP agent, and move that
+    to another agent.
+
+    :param qclient: A neutronclient
+    :param noop: Optional noop flag
+    :param agent_host: the hostname of the DHCP agent to migrate networks from
```

Comment:
Could you add comments for all parameters?

Reply:
done